apps/aechannel/src/aesc_offchain_tx.erl

```diff
@@ -56,8 +54,7 @@
           channel_id         :: aeser_id:id(),
           updates            :: [aesc_offchain_update:update()],
           state_hash         :: binary(),
-          round              :: non_neg_integer(),
-          block_hash         :: aesc_pinned_block:hash()
```

Comment:
So this was not stored in DB as record.

Reply:
It was always stored in a serialized format and never as a record